scripts/prepdocslib/listfilestrategy.py

```diff
@@ -26,6 +26,9 @@ def __init__(self, content: IO, acls: Optional[dict[str, list]] = None):
     def filename(self):
         return os.path.basename(self.content.name)
 
+    def file_extension(self):
```

Comment:
Can you add test to test_listfilestrategy.py?

Reply:
done.